input/cql/PCSBPScreeningFollowUpFHIR.cql

```diff
@@ -250,26 +251,35 @@ define "NonPharmacological Intervention Not Ordered":
     union [ServiceNotRequested: "Dietary Recommendations"]
     union [ServiceNotRequested: "Recommendation to Increase Physical Activity"]
     union [ServiceNotRequested: "Referral or Counseling for Alcohol Consumption"] ) NonPharmIntervention
-    where NonPharmIntervention.reasonRefused in "Patient Declined"
-      and NonPharmIntervention.status = 'completed'
+      where NonPharmIntervention.status = 'completed'
+        and exists(NonPharmIntervention.extension E
+          where E.url = 'http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-doNotPerformReason' 
+          and E.value as Code in "Patient Declined"
+        )
```

Comment:
I think this is expected to work as `NonPharmIntervention.reasonRefused`

Reply:
Right, extensions are surfaced as elements on the resource named according to the sliceName of the extension. So you can just say `reasonRefused`, you don't have to dig in to the extensions (though that will still work :))